### ISSUE TITLE
[Idea] Change the regular checkbox to a toggle switch

### ISSUE THREAD

**Issue by eveneme** [WoltLab/WCF#5931]
Wherever possible, change the regular standard checkbox to a toggle switch. Toggle switches are more convenient, modern, and look better. Most new components or websites use toggle switches instead of the outdated checkboxes. Please consider this proposal as time moves forward, and it would be great if WoltLab also advanced in terms of UI/UX.

Here are several advantages of using a toggle switch instead of a regular checkbox:

1. Better visibility of state:

- Toggle switch clearly shows whether an option is on or off. Its visual form and movement are more intuitive.
- Checkbox requires a closer look to check if it is marked or not.

2. Modern appearance:

- Toggle switch has a more modern and stylish look, which can make the user interface look more up-to-date and professional.
- Checkbox may seem outdated compared to newer user interface elements.

3. Intuitiveness:

- Toggle switch mimics real-life switches, making it more intuitive for users, especially in the context of turning settings on and off.
- Checkbox is less intuitive, especially for users who are not familiar with computers.

4. Ease of use on mobile devices:

- Toggle switch is usually larger and easier to tap on touch screens.
- Checkbox can be harder to select on small touch screens.

5. Aesthetics and consistency:

- Toggle switch can improve the overall aesthetics of the user interface, especially if used throughout the system for a unified look.
- Checkbox can disrupt the aesthetics, particularly in modern applications or websites.

6. Faster recognition:

- Toggle switch allows users to quickly recognize the current state of a setting due to its dynamic form.
- Checkbox requires more time to understand, especially when there are many options on one page.

All these advantages make the toggle switch preferred in modern user interface designs, offering a better user experience compared to the traditional checkbox.

**Example:**

![1](https://github.com/WoltLab/WCF/assets/85762221/4a268d83-0fac-464b-a2ec-e861cc62f74e)

**Comment by TimWolla** [comment]
Effectively a duplicate of #5788

**Comment by eveneme** [comment]
@TimWolla Yes, but this was more about the administrative panel and the 'YES' 'NO' buttons.

**Comment by eveneme** [comment]
Most people are switching from a regular checkbox to a toggle box; on mobile devices, toggle boxes are practically everywhere, and they're also common on new websites. In my opinion, a toggle box is more aesthetically pleasing, modernizes the interface, and is more practical than a regular, small checkbox.
